api-tests/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "api-tests",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Saleor API tests",
+  "main": "N/A",
```

Comment:
This will break if you `require()` this package - I think we should just create an `index.mjs` file and throw an error if someone tries to import it.

`N/A` is not a valid value

Reply:
This change has been requested by @krzysztofzuraw in the original PR: https://github.com/saleor/saleor/pull/12944/files/cea5c85485d6fb2c62886672374a8da63fa0c59d#r1209163098 
Please decide among yourselves what approach should be used in this case and let me know once decided so I will make the changes as needed.

Reply:
I removed main from the package.json https://github.com/saleor/saleor/pull/12963/commits/55969f2944e6fa0c9a35c1e30cc13ac0b3034502

Let me know if any other changes are needed please, cheers!